### ISSUE TITLE
Authorizing with fission endlessly spins

### ISSUE THREAD

**Issue by btheado** [Jermolene/tiddlywiki-on-fission#15]
When I visit https://tiddlywiki.fission.app/, the "Authorizing with fission" message endlessly spins. In my browser console I see `Uncaught (in promise) Error: Improperly formatted header value: skeleton` in webnative.js

**Comment by saqimtiaz** [comment]
I also get the following alert:

![image](https://user-images.githubusercontent.com/68092/154803710-704c60ba-74cd-4c05-ab6c-f8bf5916fc41.png)


I believe there have been breaking changes in the WNFS implementation.

**Comment by btheado** [comment]
Yes, I got the above alert a few weeks back. I failed to migrate my account and ended up creating a new one. With the new account I'm getting the reported webnative.js error. Maybe webnative.js needs upgraded in TW on fission.

**Comment by Jermolene** [comment]
Thanks @btheado @saqimtiaz. I'm making arrangements with @bmann to work on this again, and will make an update as soon as I can.
